sys-utils/irq-common.c

```diff
@@ -375,10 +375,22 @@ static inline int cmp_total(const struct irq_info *a,
 	return a->total < b->total;
 }
 
+static inline int cmp_ulong_desc(unsigned long a, unsigned long b)
+{
+	if(a < b)
+		return 1;
+	else if(a == b)
+		return 0;
+	return -1;
+}
+
 static inline int cmp_delta(const struct irq_info *a,
 		      const struct irq_info *b)
 {
-	return a->delta < b->delta;
+	int r;
```

Comment:
Could be reduced to
```c
if (a->delta != b->delta)
     return a->delta < b->delta;
return cmp_name(a, b);
```
(Or with the comparison inverted)

Reply:
Thanks, I've done so.